filebeat/docs/fields.asciidoc

```diff
@@ -2746,18 +2746,22 @@ Minor version of the operating system.
 
 --
 
-*`agent.type`*::
+*`beat.name`*::
 +
 --
 type: alias
 
+path: agent.type
```

Comment:
I find this literal output of `path:` a bit dry. I understand that this is the terminology used inside ElasticSearch.

But I think we could output something a bit more human friendly here:

```
*`beat.name`*::
+
--
type: alias

alias to: agent.type
```

Reply:
changed, PR will be updated in a minute.